### ISSUE TITLE
Learn documentation

### ISSUE THREAD

**Issue by gosha20777** [lacmus-foundation/lacmus-atlas-npu#1]
**Input**
- target models 
  - `Atlas 200 DK (v20)` (lighter, cool, and more compact)
  - `Atlas 500 - 3010` (big and hot)
- https://www.huaweicloud.com/intl/en-us/ascend/resource/Software - download page for software and compilers
- Atlas 200 DK
  - https://support.huaweicloud.com/intl/en-us/productdesc-A200dk_3000/atlas200_DK_pdes_19_0007.html - Atlas 200 dev kit v20 documentation (developer guide)
  - https://support.huaweicloud.com/intl/en-us/ascenddocument/index/atlas200.html - Atlas 200 documentation (production guide, similar as developer guide)
- Atlas 500 3010
  - https://support.huaweicloud.com/intl/en-us/ascenddocument/index/atlas500.html Atlas 500 documentation

**Output**
Write a simple guide (markdown format) with the process of converting a model (TensorFlow frozen graph .pb) into a format suitable for inference on Atlas 200 DK.
For example:
1. Take the model `xxx.pb`
2. Download software {link}
3. Read the documentation install guide {link}
3. Enter bash command `command`
etc.

You can use external links to external resources (it can be anything, not only sections of the official documentation).

In other words, the documentation of the Huawei is not as well structured as that of the Intel NPU. We need some kind of reference points describing the conversion process. When developing, we will be able to build on them, which will speed up the process.

In addition, I highly discourage installing software on any personal computers. Use docker instead.
